README.md

```diff
@@ -4,3 +4,9 @@ This repository contains the `genesis.json` for Celestia testnets.
 
 Please refer to the Celestia Docs for guides on getting started running
 your own node [here](https://docs.celestia.org)
+
+## Networks
+
+Network | celestia-app | celestia-node
+--- | --- | ---
+Arabica | [v0.10.0-rc1](https://github.com/celestiaorg/celestia-app/releases/tag/v0.10.0-rc1) | [v0.5.0-rc5](https://github.com/celestiaorg/celestia-node/releases/tag/v0.5.0-rc5)
```

Comment:
Suggestion: We def need to add sdk and core versions

Reply:
We can add them to this table but I'm curious why? Would they be more convenient than referencing the source of truth in celestia-app's `go.mod`. E.g:
1. https://github.com/celestiaorg/celestia-app/blob/124e53434685ada350162dc92133f5f83d2eb626/go.mod#L183
1. https://github.com/celestiaorg/celestia-app/blob/124e53434685ada350162dc92133f5f83d2eb626/go.mod#L185

Reply:
resolved in https://github.com/celestiaorg/networks/pull/162/commits/6fa44c71cab280a59db92669f66b5eb7c02a63cf